docs/state.md

```diff
@@ -669,6 +668,10 @@ AppStore
     updated: any,
     url: any
   },
+  userModelOptions: {
```

Comment:
Not sure if we need new entity, maybe just create `options` inside `userModel`. This way we don't need to pass another object into components

Reply:
This is for holding the available locales array. The problem is, when user clears ads history, the entire userModel object is cleared and we lose the locales.

Reply:
What would you propose? Currently on clear, userModel is set to an empty Map. My thinking was maintainability. We could use userModel for clearable items and userModelOptions for retained items. Otherwise, we'll need to make sure any future preservable items are accounted for.